src/ArgumentParser.cs

```diff
@@ -30,22 +30,28 @@ public ArgumentParser(string[] args)
           case "-g":
             SetOutputFormat(OutputType.GoogleCalendar, "Google", 40);
             break;
+	  case "--primarygoogle":
+          case "-p":
+            Console.WriteLine("WARNING: The --primarygoogle option deletes any calendars named 'My timetable'.\nIt will also input events into users' primary calendars...\n\nHit ctrl+c or close the window if you're uncertain!\n");
+            System.Threading.Thread.Sleep(12000);
+            SetOutputFormat(OutputType.PrimaryGoogle, "PrimaryGoogle", 40);
```

Comment:
The second argument is just used in a text string later.
```suggestion
            SetOutputFormat(OutputType.PrimaryGoogle, "Google", 40);
```